ERROR - 2025-10-08 06:22:10 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'lms_doraido' in C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'lms_doraido', 3306, '', 0)
#1 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MigrationRunner.php(646): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\ITE311-DORAIDO\system\Commands\Database\MigrateStatus.php(115): CodeIgniter\Database\MigrationRunner->getHistory('')
#8 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateStatus->run(Array)
#9 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:status', Array)
#10 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-DORAIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'lms_doraido' in C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MigrationRunner.php(646): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\ITE311-DORAIDO\system\Commands\Database\MigrateStatus.php(115): CodeIgniter\Database\MigrationRunner->getHistory('')
#7 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateStatus->run(Array)
#8 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:status', Array)
#9 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\ITE311-DORAIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-10-08 06:22:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'lms_doraido'
[Method: CLI, Route: migrate:status]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FROM `lms_doraido`')
 3 SYSTEMPATH\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
 4 SYSTEMPATH\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
 5 SYSTEMPATH\Database\MigrationRunner.php(646): CodeIgniter\Database\MigrationRunner->ensureTable()
 6 SYSTEMPATH\Commands\Database\MigrateStatus.php(115): CodeIgniter\Database\MigrationRunner->getHistory('')
 7 SYSTEMPATH\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateStatus->run([])
 8 SYSTEMPATH\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:status', [])
 9 SYSTEMPATH\Boot.php(388): CodeIgniter\CLI\Console->run()
10 SYSTEMPATH\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
11 ROOTPATH\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
ERROR - 2025-10-08 06:23:33 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'lms_doraido' in C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'lms_doraido', 3306, '', 0)
#1 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-DORAIDO\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-DORAIDO\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\ITE311-DORAIDO\app\Controllers\Auth.php(31): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\ITE311-DORAIDO\system\CodeIgniter.php(933): App\Controllers\Auth->login()
#8 C:\xampp\htdocs\ITE311-DORAIDO\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\ITE311-DORAIDO\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-DORAIDO\index.php(38): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'lms_doraido' in C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ITE311-DORAIDO\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ITE311-DORAIDO\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\ITE311-DORAIDO\app\Controllers\Auth.php(31): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\ITE311-DORAIDO\system\CodeIgniter.php(933): App\Controllers\Auth->login()
#7 C:\xampp\htdocs\ITE311-DORAIDO\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp\htdocs\ITE311-DORAIDO\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\ITE311-DORAIDO\index.php(38): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-10-08 06:23:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'lms_doraido'
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(31): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(38): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-10-08 06:26:07 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_doraido`.`courses`, CONSTRAINT `courses_teacher_id_foreign` FOREIGN KEY (`teacher_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', NULL, false)
#4 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-DORAIDO\app\Database\Seeds\CourseSeeder.php(70): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\Seeder.php(149): App\Database\Seeds\CourseSeeder->run()
#7 C:\xampp\htdocs\ITE311-DORAIDO\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-DORAIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-08 06:27:37 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_doraido`.`courses`, CONSTRAINT `courses_teacher_id_foreign` FOREIGN KEY (`teacher_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', NULL, false)
#4 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-DORAIDO\app\Database\Seeds\CourseSeeder.php(70): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\Seeder.php(149): App\Database\Seeds\CourseSeeder->run()
#7 C:\xampp\htdocs\ITE311-DORAIDO\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-DORAIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-08 06:33:21 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_doraido`.`courses`, CONSTRAINT `courses_teacher_id_foreign` FOREIGN KEY (`teacher_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', NULL, false)
#4 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-DORAIDO\app\Database\Seeds\CourseSeeder.php(70): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-DORAIDO\system\Database\Seeder.php(149): App\Database\Seeds\CourseSeeder->run()
#7 C:\xampp\htdocs\ITE311-DORAIDO\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-DORAIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-DORAIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-DORAIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
